bciers/package.json

```
@@ -17,7 +17,8 @@
     "reg1:e2e:ui": "nx run registration1:e2e:ui",
     "reg1:e2e:report": "nx run registration1:report",
     "reg1:coverage": "nx run registration1:test --coverage",
-    "reg1:test": "nx run registration1:test"
+    "reg1:test": "nx run registration1:test",
+    "affected:test": "nx affected:test --base=remotes/origin/main"
```

```suggestion
    "affected:test": "nx affected --base=remotes/origin/main --target=test"
```

Same as other comment, the `target` is handled by the attribute.